wiki/Project_Loved/en.md

```diff
@@ -6,16 +6,14 @@
 
 Beatmaps are moved to Loved by a nomination and voting process.
 
-Members of the Project Loved Team called **captains** regularly nominate beatmaps for a specific game mode. Almost always, these sets come from submissions from the osu! community. The current Loved submission forms and listings can be found via the following links:
+Members of the Project Loved Team called **captains** regularly nominate beatmaps for a specific game mode. These sets almost always come from submissions from the osu! community.
 
-- [osu!/osu!catch form](https://docs.google.com/forms/d/e/1FAIpQLSdbgHOVqMF8wQQKSdddW1JhC10ff6C7fb4JbEW7PBQTn9gAqg/viewform)
-- [osu!taiko form](https://docs.google.com/forms/d/e/1FAIpQLSclPWyjFByhHP45AtKD49y0RSl1TK5UOzD4dVdvjfJJQ1aCXQ/viewform)
-- [osu!mania form](https://docs.google.com/forms/d/e/1FAIpQLSeaGfoQNGMqw4qQcqRPItUZILh2fGwJR6ly6cZNY9OWPXkFhw/viewform)
-- [osu!/osu!catch listing](https://docs.google.com/spreadsheets/d/1HgHwtO3kIzT8R4ocEJMZTosADrGJRJOFL-TZI97tZS4/edit)
-- [osu!taiko listing](https://docs.google.com/spreadsheets/d/1Nk96z_xat_7ypsDF1sCTDO4i_CnHarcrCbGoTmgwHHE/edit)
-- [osu!mania listing](https://docs.google.com/spreadsheets/d/1sjkTwUSvQ5Me-6rK61rToTg2bU-yX9X29CXdzttvhtM/edit)
+To submit a beatmap for Loved, please visit the [beatmap submission page of the Project Loved website](https://loved.sh/submit) and fill out the form. Upon submitting, it will be publicly visible under the listing of its respective game mode along with other submissions. Below are the listings where these submissions can be viewed:
```

Comment:
long links = meh
also, the last sentence is kind of bulky, could it be simplified a bit (like, throw that passive voice in a bin)?
```suggestion
To submit a beatmap for Loved, please visit the [beatmap submission page](https://loved.sh/submit) of the Project Loved website and fill out the form. Upon submitting, it will be publicly visible under the listing of its respective game mode along with other submissions. Below are the listings where these submissions can be viewed:
```